app/api/resources/grpc_client.py

```diff
@@ -119,11 +119,20 @@ def get_region_checksum(self, genome_uuid: str, region_name: str):
         )
         return self.stub.GetGenomeAssemblySequenceRegion(region_checksum)
 
-    def get_dataset_attributes(self, genome_uuid: str, dataset_type: str):
+    def get_dataset_attributes(
+        self, genome_uuid: str, dataset_type: str, attribute_name: list
```

Comment:
update type hint for `attribute_name` (`list | None=None` or `Optional[int]=None`)

Reply:
I made this update @azangru , It's gonna look a bit odd in the URL but makes sense.

Reply:
Oh, I see what you mean; this will result in a url where query parameters will be `?attribute_names=foo& attribute_names=bar`. Yeah, this also feels wrong 😞

Reply:
People use both, I found one plural example so I think we're good!